Italian/guardare/vecchie-fotografie/vecchie-fotografie.md

```diff
@@ -5,3 +5,7 @@ Sembrano fotografie scattate negli anni 50, almeno a giudicare dall'abbigliament
 
 Vi sono uomini e donne di diversa età, e in alcune immagini, inspiegabilmente, vi sono delle bruciature sul viso 
 di alcune persone.
+
+[Sfogli i libri, in cerca di qualche indizio](libri-francesi/libri-francesi.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.